epoxy-processor/src/main/java/com/airbnb/epoxy/ViewAttributeInfo.kt

```diff
@@ -128,6 +152,17 @@ internal class ViewAttributeInfo(
                 super.isRequired()
             }
 
+    private fun setViewAttributeType(
+            element: Element,
+            errorLogger: ErrorLogger
+    ) {
+        when {
```

Comment:
it's cleaner to return a value than to set a field. the field should be a val to be safer